### ISSUE TITLE
RFC: Changes to the Vox Pupuli Charter

### ISSUE THREAD

**Issue by bbriggs** [voxpupuli/plumbing#83]

**Comment by bastelfreak** [comment]
Can you clarify what kind of changes you are thinking about?

**Comment by bbriggs** [comment]
These would be any changes to the actual [governing charter](https://github.com/voxpupuli/plumbing/blob/master/share/governance.md)

Given the fact that any change to that document affects how VP is governed and that the PMC is small by design, lazy consensus is not the appropriate model for changes. We can (and should) get 100% turnout easily since there are only 5 of us.

**Comment by roidelapluie** [comment]
I disagree with that. The community has chosen those rules only a few months ago, my guess is that a so big change would be really telling the community: f off...

**Comment by bbriggs** [comment]
What is your proposal for how new changes are ratified?

**Comment by roidelapluie** [comment]
In addition to that, the lazy consensus means that also the community has its voice on changes in the governance changes.

What I read in the governance paper is that everyone is part of lazy consensus and that it is someone of the PMC that clicks the 'merge' button on governance pull requests at the end.

**Comment by roidelapluie** [comment]
@bbriggs I think opening pull requests and waiting for lazy consensus ... and that it is a pmc member that merges the pull requests.

**Comment by roidelapluie** [comment]
The fact that PMC merges the PR ensures that at least one PMC member agrees with the PR.

**Comment by bbriggs** [comment]
I see. My issue with using simple lazy consensus for governance changes is the significance of changes to the governance document.

A +1 from a single community member doesn't necessarily reflect the overall feeling of the community. Likewise, a single +1 from a PMC member for a change doesn't mean the other members agree. 

What about a minimum threshold of agreement from both community and PMC? This would ensure input from community and consensus gathering as well as consensus within the PMC that the change is healthy and beneficial for the community. 

However we go about this, the role of the PMC is to serve the community, so our model for adopting change needs to reflect that.

**Comment by hunner** [comment]
We discussed this as part of today's community triage: https://github.com/voxpupuli/community-triage/blob/master/modules/notes/2016-10-20.md#discussion

To put relevant notes here:
* lazy consensus is important and should apply widely. The "mailing list" paragraph enumerates a list of like-things to which lazy consensus should apply, one of which is "policy shift" ie governance changes
* LC does not distinguish between PMC and non-PMC during lazy consensus, so votes are equal.
* "The PMC has final say over who can become a committer and will use lazy consensus for approval" implies that PMC nominates, but LC is still the whole community and required for adoption
* PMC does not (or should not) have fiat power, however there is a phrase that should be changed: "When the lazy consensus model does not converge on a decision or outcome, the PMC can step in and choose a path." This seems to imply power during non-convergence, which could be as simple as 72 hours with one dissenter and one affirmer.

**Comment by nibalizer** [comment]
Majority vote of the PMC is all that is required for governance changes. From the document:

"The project management committee has responsibilities beyond contributors that include participating in strategic planning, release planning and approving changes to the governance model."

This doesn't mean community members can't join the discussion. In the future (1-2 years) I hope we change the language so that the community has to approve governance changes. I think in the short term though we'll have plenty of obvious and noncontroversial changes and if we went to the voting population for all of them, we'd have an exhausted electorate that would simply stop voting in our elections.

**Comment by nibalizer** [comment]
Getting off-topic here, but in @hunner's comment he talked about contributor adding. According to the document:

"The PMC has final say over who can become a committer and will use lazy consensus for approval. Discussion over committer nominations will be done in private."

So basically what that means is when someone is nominated for contributor status, an email gets sent to the PMC list that says "Nominating Foo Bar for Contributor" and then after either a couple +1's or a day or two, that person gets contributor status. If the discussion coming out of it is not 'yes, add this person' then the PMC lets that person know, privately, why they aren't getting it and where they can improve.

I've personally already added someone to the contributor team (because they transferred in code) so maybe we need to revisit the policy.

**Comment by bbriggs** [comment]
> Majority vote of the PMC is all that is required for governance changes.

IMO, that should be explicit.

**Comment by bbriggs** [comment]
Maybe it already is, though. I should print the charter out and nail it to the wall by my desk ;)

**Comment by bbriggs** [comment]
> I've personally already added someone to the contributor team (because they transferred in code) so maybe we need to revisit the policy.

In practice, we have two paths for someone to become a committer: donating a project and being nominated. Maybe we say that donating code comes with an implicit nomination? If they're donating code we want to take care of, that's already a significant contribution.

**Comment by binford2k** [comment]
Vox governance appears to be fairly stable. I'm going to close this as done.
